### PR TITLE
Remove result from Redis after reading

### PR DESCRIPTION
**Pull request by booqoffsky**
Closes https://github.com/taskiq-python/taskiq-redis/issues/14

### CODE REVIEW

**Comment by s3rius** [comment]
Can you make it an option?

File: taskiq_redis/redis_backend.py [hunk 1]
```diff
@@ -75,12 +82,13 @@ async def get_result(  # noqa: WPS210
...
-                    pipe.hmget(name=task_id, keys=fields)
-                    .delete(task_id)
-                    .execute()
-                )
+            result_values = await redis.hmget(
+                name=task_id,
+                keys=fields,
+            )
+
+            if not self.keep_results:
```
[the remark this answers is not in the document]
Comment (booqoffsky) [reply taskiq_redis/redis_backend.py:90]: Did it. But maybe you should think about caching the results in the `AsyncTaskiqTask::get_result` method

[the remark this answers is not in the document]
Comment (s3rius) [reply taskiq_redis/redis_backend.py:90]: A good suggestion. Will do it in main taskiq repo.

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ from taskiq_redis.redis_backend import RedisAsyncResultBackend
 
 
 redis_async_result = RedisAsyncResultBackend(
-    url="redis://localhost:6379",
+    redis_url="redis://localhost:6379",
```
Comment (s3rius) [inline README.md:27]: Why renaming? It's correct.

**Review by s3rius — APPROVED** [review]
LGTM.

**Pull request merged**